tests/unit/adapters/admin-statistics-mail-test.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Adapter | admin statistics mail', function(hooks) {
+  setupTest(hooks);
+
+
+  test('it exists', function(assert) {
+    let adapter = this.owner.lookup('adapter:admin-statistics-mail');
```

Comment:
Most tests are like this and useless?

Reply:
Some are route matching and model creation tests. Were removed temporarily to fix up the failing CI .

Reply:
OK, so why not move all tests like this in 1 file. This is redundant

Reply:
Like Niranjan said here, these unit tests are placeholders. Remove them https://github.com/fossasia/open-event-frontend/pull/1127#issuecomment-388781161